guide/payments/overview.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: guide
+title: Transaction overview
+description: Principles to strive for while building bitcoin applications for end-users.
+nav_order: 4
+parent: Payments
+permalink: /guide/payments/overview/
+main_classes: -no-top-padding
+image: /assets/images/guide/payments/overview/page-overview.svg
+---
+
+# Transactions overview
+
+Bitcoin is a peer-to-peer push-payment system. This means that you are able to send, or *push*, bitcoin to any address at your will, at any time, without passing through a third party.
+
+
+This is radically different from the legacy financial system where it is often possible for others to *pull* and withdraw money from your account (utility companies, financial institutions, merchants etc.). When you make a payment it will pass through systems that might delay, control or block the payment. 
```

Comment:
```suggestion
This is radically different from the legacy financial system, where it is often possible for others to *pull* and withdraw money from your account (utility companies, financial institutions, merchants, etc.). When you make a payment, it will pass through systems that might delay, control or block the payment. 
```